Set OwnsImage to true when an image is set

jmarc-l opened the pull request:
Signed-off-by: Jean-Marc Levecque jmarc.levecque@bbox.fr

jmarc-l committed:
Set OwnsImage to true when an image is set

Signed-off-by: Jean-Marc Levecque <jmarc.levecque@bbox.fr>

graemeg commented:
Hi Jean-Marc. Sorry for the delay on Pull request #33 and this one. I'm actually trying to resolve a bigger problem which causes many memory leaks - relating to the TfpgImageList. It might affect the ImagePanel widget too. I'm almost done with my tests and code changes, then I'll action this pull request.

jmarc-l commented:
Hi Graeme,

No hurry :)
I added this to avod a memory leak because OwnsImage wasn't set, so at call of 
destroy the image was not freed.

I added today another branch called Themes with a few commits to allow my 
themes to work. Attached is a demo of these themes. I think they are ready to 
be published. I already used them in some applications, mainly Unimesur.

Best regards
Jean-Marc

Le lundi 5 octobre 2015, 05:44:20 Graeme Geldenhuys a écrit :

> Hi Jean-Marc. Sorry for the delay on Pull request #33 and this one. I'm
> actually trying to resolve a bigger problem which causes many memory leaks
> - relating to the TfpgImageList. It might affect the ImagePanel widget too.
>   I'm almost done with my tests and code changes, then I'll action this pull
>   request.
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/graemeg/fpGUI/pull/35#issuecomment-145516275

graemeg commented:
As explained in the _fpgui.support_ newsgroup. The behaviour is as designed.
